enterprise/app/root/root.css

```
@@ -21,8 +21,15 @@ body {
   box-shadow: 0 -1px 1px rgba(0, 0, 0, 0.12) inset;
 }
 
-.impersonation-toolbar button {
+.impersonation-toolbar .exit-button {
   margin-left: auto;
+}
+
+.impersonation-toolbar .generate-api-key-button {
+  margin-left: 30px;
```

uber nit: spacing values are typically multiples of 8px

```suggestion
  margin-left: 32px;
```

---

done